markdown/developers/gitpod/gitpod_introduction.md

```diff
@@ -13,7 +13,7 @@ Gitpod is free (with a GitHub/GitLab/Bitbucket account) for use up to 50hours/mo
 
 > Gitpod has kindly recognised nf-core as an open-source project, so if you are a member of the [nf-core GitHub organisation](https://github.com/nf-core/) (ask on Slack in the `#github-invitations` channel) then you will automatically be moved onto the _'Professional Open Source plan'_ with unlimited hours.
 
-Gitpod runs in your browser (recommended with Chrome, Firefox or Edge). Their website has [extensive documentation](https://gitpod.io/).
+Gitpod runs in your browser (recommended with Chrome, Firefox or Edge). Their website has [extensive documentation](https://www.gitpod.io/docs/introduction).
```

Comment:
I see that https://www.gitpod.io/docs automatically redirects to https://www.gitpod.io/docs/introduction but I wonder if it isn't better to link https://www.gitpod.io/docs instead so that it's less likely that the link will break in the future if they decide to change this redirect. I think it makes sense to expect that /docs will always point to the docs  😬

Reply:
Let me know what you think. If you think it's OK the way it is, just resolve this conversation and I'll merge 😉